ocis-pkg/version/version.go

```diff
@@ -16,7 +16,7 @@ var (
 	// LatestTag is the latest released version plus the dev meta version.
 	// Will be overwritten by the release pipeline
 	// Needs a manual change for every tagged release
-	LatestTag = "3.1.0-rc.1+dev"
+	LatestTag = "4.0.0"
```

Comment:
```suggestion
	LatestTag = "4.0.0+dev"
```

Reply:
The final release tag is overwritten by the build. Please always keep the `+dev` meta version. It needs to be there for local dev builds. All CI builds overwrite the LatestTag with the real git version.